### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Corpus has been renamed to testsuite.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+image: ghcr.io/capuanob/xml-c-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Tests has been renamed to testsuite. You may also omit the image name.

File: fuzz/fuzz.c [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
```
Comment (xansec) [inline fuzz/fuzz.c]: Can we rename this so that it is clear what's being fuzzed?

File: test/CMakeLists.txt [hunk 4]
```diff
@@ -15,6 +15,14 @@ target_compile_options(
 		-std=c11
```
Comment (xansec) [inline test/CMakeLists.txt:15]: Is it necessary to make changes to all of the CMakeLists files?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: test/CMakeLists.txt (hunk 4 above)
Comment (capuanob) [reply test/CMakeLists.txt:15]: I needed to do this in order to fully instrument the library for fuzzing. If I omitted this, I wasn't getting proper instrumentation across the codebase

**Comment by capuanob** [comment]
@xansec Made the requested changes! Look forward to your feedback

**Comment by xansec** [comment]
Removed git clone. This should be ok now.

**Pull request merged**